README.md

```diff
@@ -5,7 +5,7 @@ This is a work-in-progress. It's alpha quality. Feedback appreciated.
 # Why?
 
 The Ember addons ecosystem is one of the best things about Ember. But
-users stuck on legacy build systems have a hard time taking advantage
+users of non ember-cli build systems have a hard time taking advantage
```

Comment:
I'd just want to remove the word "yet". I wanted to take away the implication that not using ember-cli was somehow incorrect.

> But if you're not using ember-cli, taking advantage of them has historically been tough.

...maybe?